addons/project/models/project_milestone.py

```diff
@@ -28,7 +28,7 @@ def _get_default_project_id(self):
     is_deadline_future = fields.Boolean(compute="_compute_is_deadline_future")
     task_count = fields.Integer('# of Tasks', compute='_compute_task_count', groups='project.group_project_milestone')
     done_task_count = fields.Integer('# of Done Tasks', compute='_compute_task_count', groups='project.group_project_milestone')
-    can_be_marked_as_done = fields.Boolean(compute='_compute_can_be_marked_as_done', groups='project.group_project_milestone')
+    can_be_marked_as_done = fields.Boolean(compute='_compute_can_be_marked_as_done')
```

Comment:
Because it is used by invisible, required... which do not have a group. Therefore either all should have the group or none. Here, it does not seem to me a lack of security to leave it accessible.